setup.cfg

```diff
@@ -73,6 +73,10 @@ testing =
     pytest-forked
     pytest-icdiff
     robot-descriptions
+viz =
+    mediapy
+    mujoco >= 3.0.0
 all =
     %(style)s
     %(testing)s
+    %(viz)
```

Comment:
```suggestion
    %(viz)s
```

Reply:
Yep CI fas failing due to this. Squashed in https://github.com/ami-iit/jaxsim/pull/83/commits/5561c7ca0b4109daad5107fefce5c5a6f2a328f3.